packages/core/src/render3/util/global_utils.ts

```diff
@@ -40,6 +41,7 @@ let _published = false;
 export function publishDefaultGlobalUtils() {
   if (!_published) {
     _published = true;
+    publishGlobalUtil('ɵsetProfiler', setProfiler);
```

Comment:
I would also propose adding a warning here:

```suggestion
    /**
     * Warning: this function is *INTERNAL* and should not be relied upon in application's code.
     * The contract of the function might be changed in any release and/or the function can be removed completely.
     */
    publishGlobalUtil('ɵsetProfiler', setProfiler);
```

Would it be more convincing to call it something like `__setProfiler__`? :)

Reply:
I kept it `ɵsetProfiler` for consistency with other APIs we use between tooling and framework.

Reply:
Ok, makes sense.

What do you think of adding extra warnings into the code that the `ɵsetProfiler` should not be used in apps code (as mentioned in my comment above)?